example-schema/relationships/CuriousSibling.yaml

```diff
@@ -1,9 +1,11 @@
 name: CuriousSibling
 from:
   type: MainType
+  otherNodeName: youngerSibling
```

Comment:
`otherNodeName` doesn't feel like the right name for this concept as its only applicable when the `from` and `to` types are the same.  Something like `selfReferenceAlias` would be better but thats still not right.

Reply:
I looked at the GrandStack docs and maybe `...FieldName` would be good

`unambiguousFieldName` is the reason for setting it, but that's a bit off

either way, we can use it as a working title and do a big find and replace if we come up with better